conf/peers.xml

```diff
@@ -44,6 +44,9 @@
  	<!-- you can specify an even rtp port to use for incoming and outgoing rtp
  	     traffic, 0 can be used to choose a random even free port -->
   <rtpPort>0</rtpPort>
+  <!-- if you need to specify an authorization username different than
+       the userPart -->
```

Comment:
Hey thanks for getting back so quickly. I updated that line to include some information about the default behavior.